srcpkgs/python3-tkinter/template

```diff
@@ -71,25 +69,31 @@ do_install() {
 
 	# We only care for the idle and tkinter modules.
 	vmkdir usr/bin
-	vmkdir usr/lib/python${version%.*}/lib-dynload
-	mv ${wrksrc}/tmp-destdir/usr/bin/idle${version%.*} \
-		${DESTDIR}/usr/bin/idle${version%.*}
+	vmkdir "${py3_lib}/lib-dynload"
+
+	local _idle="idle${version%.*}"
+	local _src_py_lib="${wrksrc}/tmp-destdir/${py3_lib}"
+	local _dst_py_lib="${DESTDIR}/${py3_lib}"
+
+	mv "${wrksrc}/tmp-destdir/usr/bin/${_idle}" "${DESTDIR}/usr/bin"
+	mv "${_src_py_lib}/lib-dynload"/_tkinter.*.so "${_dst_py_lib}/lib-dynload"
+
 	for lib in idlelib tkinter turtledemo turtle.py; do
-		mv ${wrksrc}/tmp-destdir/usr/lib/python${version%.*}/${lib} \
-			${DESTDIR}/usr/lib/python${version%.*}/
+		mv "${_src_py_lib}/${lib}" "${DESTDIR}/${py3_lib}"
 	done
-	mv ${wrksrc}/tmp-destdir/usr/lib/python${version%.*}/lib-dynload/_tkinter.cpython-*.so \
-		${DESTDIR}/usr/lib/python${version%.*}/lib-dynload/
-	ln -sf idle${version%.*} ${DESTDIR}/usr/bin/idle3
+
+	ln -sf "${_idle}" ${DESTDIR}/usr/bin/idle3
+	ln -sf "${_idle}" ${DESTDIR}/usr/bin/idle
 }
 
 idle-python3_package() {
 	short_desc="${_desc} - IDE for Python3 using Tkinter"
-	pycompile_dirs="usr/lib/python${version%.*}/idlelib"
+	pycompile_dirs="${py3_lib}/idlelib"
 	depends="${sourcepkg}-${version}_${revision}"
+	conflicts="python<3.0.0_1 python2<=2.7.18.11_1 python3<=3.13.1_1"
```

Comment:
I would put: `depends="python>=3.0.0_1 python3>=3.13.1_2"`
conflicts resolution works before version resolution, install them first them is nearly impossible, afaik.

Reply:
A dependency on `python` is not appropriate, because the these packages should function properly without installing or ignoring the convenience metapackage. The conflict is only to handle cases where people try to update `python3-tkinter` while locking `python` at 2.7, before it was a metapackage, in which case this update will overwrite the alternative symlink I'm now removing.